template/Pages/Home/styles.js

```diff
@@ -92,6 +93,9 @@ export const StepsBlock = styled.div`
       text-align: center;
     }
   }
+  @media (max-width: 790px) {
```

Comment:
@CarlosOrdering we're using mobile first in design, pls use `@media (min-width)`

Reply:
I make the modification in the last commit